File: alphabit-coding-challenge-03/is_it_a_good_team/submissions/accepted/survey_corps.go
package main

import "fmt"

func main() {
	var m string
	var res []int
	fmt.Scanf("%v", &m)
	l := len(m)
	for i := 0; i < l; i++ {
		if string(m[i]) == "l" {
			res = append(res, i)
		}
	}
	err := res[0]
	for i := 1; i < len(res); i++ {
		err ^= res[i]
	}
	fmt.Printf("%v", err)

}

File: alphabit-coding-challenge-03/is_it_a_good_team/solutions/survey_corps.pas
program surveycorps;

var
  m : string ;
  l ,i ,j , err: integer ;
  res :array [0..10000] of integer ;
begin
  readln(m);
  l := length(m);
  j:=0;
  for i :=0 to l do
  begin
    if m[i]='l' then
       begin
       res[j] := i-1;
       j := j+1;
       end;
  end;
  err := res[0];
   for i :=1 to j do
   begin
     err := err xor res[i];
   end;

   write(err);
   readln();
end.

